spring-amqp/src/test/java/org/springframework/amqp/support/converter/Jackson2JsonMessageConverterTests.java

```diff
@@ -50,13 +54,14 @@
 public class Jackson2JsonMessageConverterTests {
```

Comment:
I don't understand - there's an autowired field - the config is in `src/test/resources` and it runs in 0.13 seconds for me.